R/fonts.R

```diff
@@ -441,7 +443,12 @@ font_dep_google_local <- function(x) {
       x$cache$remove(css_key)
       return(font_dep_google_local(x))
     }
-    download_file(url, f)
+    if (!informed) {
+      rlang::inform(paste0("Downloading google font ", x$family, " to local cache"))
```

Comment:
Unfortunately, the `to local cache` part isn't always true.

`x$cache` can be a "mock" cache that never actually caches. I think, to workaround this, it could make sense for `resolve_cache()` to return something like a `mock_cache = TRUE` field [here](https://github.com/rstudio/sass/pull/141/files#diff-351b5dab7b01d1f0771f60ef93ab62a8e10a1656eff2537e31456c9b7339dde8R406) which we'd then use for constructing the message.

Also, when a cache isn't a mock cache, I suppose it could also make sense to include `x$cache$dir()` in the message (so the user knows where those files are going)?

Reply:
Let me know if these changes are what you are thinking.

Reply:
I went ahead and made the change I was thinking of in  [cbfd605](https://github.com/rstudio/sass/pull/141/commits/cbfd605fd7a6ce7b711dd5090a87663766c735d0). Once I have your 👍 , I'll merge.